content/docs/use-cases/ci-cd-for-machine-learning.md

```diff
@@ -0,0 +1,69 @@
+# Continuous Integration and Deployment for Machine Learning
+
+MLOps and DataOps apply DevOps methodologies to machine learning (ML) and data
+management. The fields are rapidly evolving beyond a mere set of best practices,
+and now encompass the entire product lifecycle management. A core component is
+the automation of testing and deployment: CI/CD (continuous integration and
+continuous delivery).
+
+CI/CD is easy to set up with practically any Git repository. Some example
+scenarios are:
+
+- Agile development
+  - Automating quality assurance (QA) means teams can confidently deploy new
+    versions several times a day and even before the weekend without fear of
+    bugs/regressions
+- Easy testing
+  - When new data is pushed in a pull request (PR), validation tests
+    automatically run in the cloud (without each collaborator setting up
+    potentially inconsistent local testing environments)
+  - When a new model is pushed in a PR, reports (on performance against the
+    current deployed models) are automatically generated and posted as a PR
+    comment with graphs and tables
+- Continuous machine learning (CML): online training
+  - Schedule regular jobs pulling in data updates from public sources to refine
+    deployed models
+  - Provision hardware in the cloud and run a brute force hyperparameter search
+
+In ML, CI/CD can be used to automatically build, test and push data and models
+to production in the cloud with minimal effort. There are many components, such
+as:
+
+1. Data collection & storage
+2. Metadata management
+3. Validation & testing (including provisioning resources)
+4. Sharing & deployment
+5. Monitoring & feedback
+
+Quite often, CI/CD for ML is hard to set up and debug, and solutions unclear.
+DVC can alleviate most (in not all) of the
+[hidden technical debt in ML](https://papers.nips.cc/paper/2015/file/86df7dcfd896fcaf2674f757a2463eba-Paper.pdf).
```

Comment:
Is this really true, and how does it connect to the components listed above? For example, I'm not sure I've seen dvc used for monitoring and feedback.

This seems sort of like the thesis statement of this use case, and IMO it would be nice to be more specific about DVC's strengths without getting too details. WDYT about something like this: "DVC provides lightweight tools that can work on any infrastructure and can alleviate hidden technical debt by managing data, models, and code together."